File: code_wars/8kyu/number_to_reversed_array_digits/array_digits.go
package kata

func Digitize(n int) []int {
	result := []int{}

	if n == 0 {
		result = append(result, 0)
		return result
	}

	for ; n > 0; n = n / 10 {
		result = append(result, n%10)
	}

	return result
}

File: code_wars/8kyu/number_to_reversed_array_digits/array_digits.py
def digitize(n: int) -> list[int]:
    if n == 0:
        return [0]
    result = []
    while n > 0:
        result.append(n % 10)
        print(f"rest:{n%10}")
        print(f"floor div: {n//10}")
        n = n // 10
    return result


print(digitize(1234))
